circleci/images/exttester/Dockerfile

```diff
@@ -87,6 +87,7 @@ RUN apt-get update \
     cpanminus \
     curl \
     debian-archive-keyring \
+    expect \
```

Comment:
```suggestion
```

removing expect as discussed in https://github.com/citusdata/pgazure/pull/167#discussion_r1144471747